### ISSUE TITLE
Retiring this repo

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-forge-maintenance#50]
Would like to suggest we archive this repo. It held content that was useful early on for pinning, re-rendering, etc. Though we largely handle this through [conda-forge-webservices]( https://github.com/conda-forge/conda-forge-webservices ) and conda-build 3 these days. AFAIK we are not using the maintenance webservices at all. As a result the functionality here doesn't seem relevant today. Though the history may be. What do people think about this?

cc @conda-forge/core

**Comment by scopatz** [comment]
Seems fine to me

**Comment by jcfr** [comment]
Makes sense. To avoid confusion, I also suggest to "unpin" the repo from the GitHub organization.

**Comment by scopatz** [comment]
I actually can't unpin the repo.  The github UI never loads it

**Comment by jakirkham** [comment]
Yeah I agree it would be nice to unpin. Unfortunately the UI might be broken here (trying to load all repos in conda-forge?). We should try to raise this issue to GitHub if possible.

**Comment by jcfr** [comment]
As a side note, I looked for an [API endpoint](https://developer.github.com/v3/) but didn't find one.

> We should try to raise this issue to GitHub if possible

Each time I had to contact GitHub, they responded quickly

Would like me to reach out ?

**Comment by jakirkham** [comment]
Sure, if you would like, that would be very helpful. Feel free to cc me.

**Comment by jakirkham** [comment]
Have gone through the issues in this repo. Explained how we addressed any issues or noted this is not maintained and closed them. Will leave this issue open and pinned for context and archive this repo.
